torch/csrc/jit/runtime/instruction.h

```diff
@@ -50,7 +50,6 @@ namespace jit {
   _(DICT_CONSTRUCT, "TI") /* construct a dict of type X, using N inputs */     \
   _(CREATE_OBJECT, "T") /* create an object of type X */                       \
   _(ISINSTANCE, "TI") /* check object is one of  types[X:X+N]  */              \
-  _(TUPLE_SLICE, "II") /* slice tup[X:(X+N)] */                                \
```

Comment:
We should probably talk with Martin Yuan on mobile team to see if he needs to mimic this change.

Reply:
@iseeyuan could you take a look?

Reply:
It might affect the backward compatibility of mobile. If an old mobile model with TUPLE_SLICE is loaded by this code, it will fail. If the model is already deployed by production, there would be a regress. Is it possible to keep TUPLE_SLICE to make it backward compatible?